src/s_sound.c

```diff
@@ -49,7 +49,7 @@ typedef struct channel_s
 // the set of channels available
 static channel_t channels[MAX_CHANNELS];
 // [FG] removed map objects may finish their sounds
-static degenmobj_t sobjs[MAX_CHANNELS];
+static mobj_t sobjs[MAX_CHANNELS];
```

Comment:
Any objection to using `mobj_t` for "unlinked sounds"?

Reply:
Not a real problem, but we could stay with `degenmobj_t` and have its `thinker` field point to `P_DegenMobjThinker()`, which is introduced by @rfomin in the other PR?

Reply:
Would that preserve `sobj->info->actualheight`? That value is needed here:

https://github.com/fabiangreffrath/woof/blob/1d644070d8b7464792536f41ddd893df32d4f7bb/src/i_3dsound.c#L173-L190

Reply:
Ha, nope. 😁